mentoring/programs/README.md

```diff
@@ -1,2 +1,8 @@
-future home of mentoring programs, how to get involved, and other info for
-all contributors.  
+# Home of the Kubernetes Mentoring Program  
+
+## Meetings
+
+APAC friendly time is, Monday at 20:30 am PT, every second and fourth week of the month
+EU/NA friendly time is, Tuesdays at 08:30 pm PT, every second and fourth week of the month
```

Comment:
nit: we can remove the am and pm from here since we use the 24 hour format

Reply:
Corrected